src/persist-client/src/lib.rs

```diff
@@ -123,6 +123,33 @@ impl PersistLocation {
             .await?;
         Ok((blob, consensus))
     }
+
+    /// Opens the associated implementations of [BlobMulti] and [Consensus], in a wrapper that
+    /// induces unreliability.
+    pub async fn open_unreliable(
```

Comment:
I have a mild preference for the way this is done in the maelstrom PR over this, but I'm willing to change my mind. thoughts? 

https://github.com/MaterializeInc/materialize/pull/12108/commits/aa4bfcdc2dd4580b6e315761ad36a6c47da61d4c

Reply:
I have no preference. I can remove the unreliable code from the example and maybe add it back once your PR is in.